README.md

```diff
@@ -283,10 +283,19 @@ The integration tests are hard-coded to hit `http://localhost:18080`. In the fut
 
 > **Warning:** Running the integration tests wipes out the entire `db.poet.works` collection and inserts testing data. This is done by the `test/integration/PrepareDB.ts` file. In the future, a less invasive mechanism will be developed. Meanwhile, make sure you're comfortable with this before running the integration tests!
 
-Currently, Po.et Node is lacking some tests. The most critical paths that aren't being tested right now are:
+Functional tests are run as follows:
 
-- Broadcasting of transactions in a single Node (submit work, wait a bit, get work and expect transactionId to be set and valid)
-- Replication across nodes (submit WORK to Node A, get WORK in Node B)
+  - Setup a docker-compose override file
+```bash
+  $ cp docker-compose.override.yml.example docker-compose.override.yml
+```
+  - Edit the new file and uncomment the 'bitcoind-2' service
```

Comment:
Which lines should be uncommented exactly?

```
  bitcoind-2:
    # image: ruimarinho/bitcoin-core:0.16.3
    # volumes:
    #   - bitcoind2:/home/bitcoin/.bitcoin
    # ports:
    #   - "19443:18443"
    # command:
    #   -printtoconsole
    #   -testnet=${USE_BTC_TESTNET:-0}
    #   -regtest=${USE_BTC_REGTEST:-1}
    #   -rpcuser=bitcoinrpcuser
    #   -rpcpassword=bitcoinrpcpassword
    #   -rpcallowip=::/0
    #   -server
    #   -addnode=bitcoind-1
    # links:
    #   - bitcoind-1:bitcoind-1
```

Everything except for:

```
    # volumes:
    #   - bitcoind2:/home/bitcoin/.bitcoin
```

?